### ISSUE TITLE
SCAM, DONT USE.

### ISSUE THREAD

**Issue by Xtendera** [StaffBesting/UnlockAllLoader#1]
It is most definatly a scam. If u ran the exe, then run your antivirus software. It is almost certainly a RAT.

**Comment by Aethese** [comment]
Of course it is. Just an exe as the source is a dead giveaway. The fact they think they can get away with this is hilarious.

**Comment by Xtendera** [comment]
Exactly. but I know there are people desperate enough to look good in these games to try these free cheats. Don't do it people.
